.github/workflows/linter.yml

```diff
@@ -48,3 +48,4 @@ jobs:
           mypy --config-file=.github/linters/mypy.ini data_server/
           mypy --config-file=.github/linters/mypy.ini run_ingestion/ python/ingestion/
           mypy --config-file=.github/linters/mypy.ini run_gcs_to_bq/ python/ingestion/
+          mypy --config-file=.github/linters/mypy.ini dags/
```

Comment:
Does the path now need to be updated here?

Reply:
yes. good catch